.buildkite/publish/build-multiarch-docker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -exu
+set -o pipefail
+
+function realpath {
+  echo "$(cd "$(dirname "$1")"; pwd)"/"$(basename "$1")";
+}
+
+SCRIPT_DIR=$(realpath "$(dirname "$0")")
+BUILDKITE_DIR=$(realpath "$(dirname "$SCRIPT_DIR")")
+PROJECT_ROOT=$(realpath "$(dirname "$BUILDKITE_DIR")")
+
+VERSION_PATH="$PROJECT_ROOT/connectors/VERSION"
+VERSION=$(cat $VERSION_PATH)
+
+if [[ "${USE_SNAPSHOT:-}" == "true" ]]; then
+  echo "Adding SNAPSHOT labeling"
+  VERSION="${VERSION}-SNAPSHOT"
+fi
+
+BASE_TAG_NAME="docker.elastic.co/enterprise-search/elastic-connectors"
+TAG_NAME="${BASE_TAG_NAME}:${VERSION}"
+AMD64_TAG="${BASE_TAG_NAME}-amd64:${VERSION}"
+ARM64_TAG="${BASE_TAG_NAME}-arm64:${VERSION}"
+
+# ensure +x is set to avoid writing any sensitive information to the console
+set +x
+
+VAULT_ADDR=${VAULT_ADDR:-https://vault-ci-prod.elastic.dev}
+VAULT_USER="docker-swiftypeadmin"
+DOCKER_USER=$(vault read -address "${VAULT_ADDR}" -field user_20230609 secret/ci/elastic-connectors/${VAULT_USER})
+DOCKER_PASSWORD=$(vault read -address "${VAULT_ADDR}" -field secret_20230609 secret/ci/elastic-connectors/${VAULT_USER})
+
+echo "Logging into docker..."
+buildah login --username="${DOCKER_USER}" --password="${DOCKER_PASSWORD}" docker.elastic.co
+
+echo "Creating manifest..."
+buildah manifest create $TAG_NAME \
```

Comment:
oh this is cool. I didn't know about `buildah manifest`. Do you know what the tradeoffs are between this and using drivah with their `toml` syntax? Should we switch the data-extraction-service to use `buildah manifest` instead?

Reply:
That I do not... I will look into it though!

Reply:
... and I don't know if `drivah` can create multi-arch images... :/.